unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -625,21 +647,25 @@ public VoxelEntityHit GetCloserUnselectedVoxelEntityOnPointer()
         VoxelEntityHit voxelEntityHit = null;
 
         hits = Physics.RaycastAll(ray, RAYCAST_MAX_DISTANCE, layerToRaycast);
+
         foreach (RaycastHit hit in hits)
         {
             string entityID = hit.collider.gameObject.name;
 
             if (sceneToEdit.entities.ContainsKey(entityID))
             {
                 DCLBuilderInWorldEntity entityToCheck = builderInWorldEntityHandler.GetConvertedEntity(sceneToEdit.entities[entityID]);
+
                 if (entityToCheck == null) continue;
 
+                Camera camera = Camera.main;
```

Comment:
I think it's better to declare de `Camera` variable as a private variable of the class and move this assignation out of the loop (`Awake`, `Start`...?) in order to execute it only once time, don't you?

Reply:
This actually falls out of the scope of this PR. I just couldn't bear the repeated use of `Camera.main` and and kinda mitigated it. We should make this a private member for sure, but I think its better to do it in another PR.